src/components/calcite-notice/calcite-notice.scss

```diff
@@ -1,3 +1,13 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-notice-spacing-token-large: the amount of left padding next to notice content
```

Comment:
These seem to be for internal use, we should expose the spacing.

Reply:
>we should expose the spacing

Can you clarify?

Reply:
Should not * , whoops. Editing original.